gradle-plugin/src/main/kotlin/dev/schlaubi/mikbot/gradle/TransientDependencies.kt

```diff
@@ -1,3 +1,3 @@
 package dev.schlaubi.mikbot.gradle
 
-const val transientDependencies = "dev.schlaubi.mikbot:api\ncom.kotlindiscord.kord.extensions:unsafe\ncom.kotlindiscord.kord.extensions:kord-extensions\nio.insert-koin:koin-logger-slf4j\nio.insert-koin:koin-core-jvm\ndev.schlaubi:annotations\ndev.kord.x:emoji\ndev.kord:kord-core\norg.litote.kmongo:kmongo-coroutine-serialization\norg.litote.kmongo:kmongo-coroutine-core\ndev.kord:kord-rest\ndev.kord:kord-voice\ndev.kord:kord-gateway\ndev.kord:kord-common\ndev.kord.cache:cache-map\ndev.kord.cache:cache-api\norg.jetbrains.kotlinx:kotlinx-coroutines-reactive\ndev.schlaubi:stdx-full\ndev.schlaubi:stdx-coroutines-jvm\nio.ktor:ktor-client-cio-jvm\nio.ktor:ktor-serialization-kotlinx-json-jvm\nio.ktor:ktor-client-content-negotiation-jvm\nio.ktor:ktor-client-websockets-jvm\nio.ktor:ktor-http-cio-jvm\nio.ktor:ktor-network-tls-jvm\nio.ktor:ktor-network-jvm\nio.ktor:ktor-client-core-jvm\nio.ktor:ktor-serialization-kotlinx-jvm\nio.ktor:ktor-websocket-serialization-jvm\nio.ktor:ktor-serialization-jvm\nio.ktor:ktor-websockets-jvm\nio.ktor:ktor-events-jvm\nio.ktor:ktor-http-jvm\nio.ktor:ktor-utils-jvm\norg.jetbrains.kotlinx:kotlinx-coroutines-slf4j\nio.ktor:ktor-io-jvm\norg.jetbrains.kotlinx:kotlinx-coroutines-core-jvm\norg.jetbrains.kotlinx:kotlinx-coroutines-jdk8\ncom.kotlindiscord.kord.extensions:annotations\ncom.kotlindiscord.kord.extensions:token-parser\ndev.schlaubi:stdx-logging-jvm\nio.github.microutils:kotlin-logging-jvm\norg.litote.kmongo:kmongo-serialization-mapping\ncom.github.jershell:kbson\ndev.schlaubi:stdx-core-jvm\ndev.schlaubi:stdx-envconf-jvm\ndev.schlaubi:stdx-serialization-jvm\norg.jetbrains.kotlin:kotlin-stdlib-jdk8\norg.pf4j:pf4j-update\norg.pf4j:pf4j\nio.sentry:sentry\ncom.google.code.gson:gson\norg.ow2.asm:asm\norg.litote.kmongo:kmongo-async-shared\norg.litote.kmongo:kmongo-property\norg.litote.kmongo:kmongo-shared\norg.litote.kmongo:kmongo-id\norg.litote:kreflect\norg.jetbrains.kotlin:kotlin-reflect\norg.mongodb:mongodb-driver-reactivestreams\norg.jetbrains.kotlinx:kotlinx-datetime-jvm\ncommons-validator:commons-validator\nio.projectreactor:reactor-core\norg.reactivestreams:reactive-streams\ncommons-beanutils:commons-beanutils\ncommons-collections:commons-collections\norg.jetbrains.kotlinx:kotlinx-serialization-json-jvm\norg.jetbrains.kotlinx:kotlinx-serialization-core-jvm\norg.mongodb:mongodb-driver-core\norg.mongodb:bson\norg.jetbrains.kotlin:kotlin-stdlib-jdk7\norg.jetbrains.kotlinx:atomicfu-jvm\norg.litote.kmongo:kmongo-data\norg.jetbrains.kotlin:kotlin-stdlib\norg.jetbrains.kotlin:kotlin-stdlib-common\ncommons-logging:commons-logging\ncommons-codec:commons-codec\ncom.github.zafarkhaja:java-semver\ncom.ibm.icu:icu4j\norg.jetbrains:annotations\ncommons-digester:commons-digester\nch.qos.logback:logback-classic\norg.slf4j:slf4j-api\nch.qos.logback:logback-core"
```

Comment:
Why was this updated

Reply:
I actually don't know

Reply:
you need to call `rebuild-plugin-dependency-list.ps1` for this to happen